core/includes/config.inc

```diff
@@ -1248,9 +1248,9 @@ class ConfigFileStorage implements ConfigStorageInterface {
    */
   public function initializeStorage() {
     if (!file_prepare_directory($this->directory, FILE_CREATE_DIRECTORY | FILE_MODIFY_PERMISSIONS)) {
-      throw new ConfigStorageException(format_string('The directory %directory could not be created or could not be made writable. ', array(
-      '%directory' => $config_directory,
-    )));
+      throw new ConfigStorageException(format_string("The directory '%directory' could not be created or could not be made writable.", array(
```

Comment:
The "%" already displays as `<em>` tags so I think the quotes are unnecessary. Otherwise good.

Reply:
We use italics (via `%`) when referring to names of things. When referring to paths and code, we use regular text (`@`) and wrap it around `<code>` tags:

```suggestion
      throw new ConfigStorageException(format_string('The directory <code>@directory</code> could not be created or could not be made writable.', array(
```

Reply:
> We use italics (via %) when referring to names of things. When referring to paths and code, we use regular text (@) and wrap it around <code> tags:

Really? Where is that documented?

Reply:
I did try just italics first, but it didn't really stand out that much from the rest of the text and so wasn't obvious. I'll use `<code>` instead.